modules/dnn/src/dnn.cpp

```diff
@@ -4531,7 +4531,6 @@ Ptr<Layer> LayerFactory::createLayerInstance(const String &type, LayerParams& pa
     cv::AutoLock lock(getLayerFactoryMutex());
     String type_ = type.toLowerCase();
     LayerFactory_Impl::const_iterator it = getLayerFactoryImpl().find(type_);
-
```

Comment:
please avoid extra changes (adding or removing new  empty lines)

Reply:
Done locally, will push in the next commits.